### ISSUE TITLE
JavaScript heap out of memory during `npm search hyper`

### ISSUE THREAD

**Issue by aem** [zeit/hyper#1150]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: 
macOS Sierra (10.12.1 - 16B2555)

- **Hyper.app version**:
1.0.0 (1.0.0.1303)

- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**:
Have only changed font size in the gist and that's all, but here you go https://gist.github.com/aem/719779aa772b49f36d7ed87177022fbe

- **Relevent information from devtools** _(CMD+SHIFT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: 
I see the following messages repeated a couple of times:
```
Unimplemented DEC Private Mode: 1034
Unknown OSC code: "11"
```
- **The issue is reproducible in vanilla Hyper.app**: 
Is Vanilla

## Issue
Quite simple, I tried to run an `npm search hyper` to browse plugins, as recommended in the README (I just downloaded Hyper a few minutes ago) and got the following error:

```sh
$ npm search hyper
⸨░░░░░░░░░░░░░░░░░░⸩ ⠴ : info using node@v6.2.0
<--- Last few GCs --->

   27318 ms: Mark-sweep 1029.8 (1434.6) -> 1025.8 (1440.1) MB, 1184.1 / 0 ms [allocation failure] [GC in old space requested].
   28617 ms: Mark-sweep 1025.8 (1440.1) -> 1025.3 (1440.1) MB, 1298.9 / 0 ms [last resort gc].
   29855 ms: Mark-sweep 1025.3 (1440.1) -> 1025.3 (1440.1) MB, 1237.8 / 0 ms [last resort gc].


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x5f749bc9e59 <JS Object>
    1: /* anonymous */ [/usr/local/lib/node_modules/npm/lib/cache/update-index.js:~90] [pc=0x19ae9fa42dcb] (this=0x5f749be4d49 <JS Global Object>,p=0x12bbd030a1c1 <String[6]: 174757>)
    2: arguments adaptor frame: 3->1
    3: InnerArrayForEach(aka InnerArrayForEach) [native array.js:~946] [pc=0x19ae9fc9755f] (this=0x5f749b04189 <undefined>,br=0x3488f7d2f7e1 <JS Function (SharedFunctionInfo 0x7...

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
Abort trap: 6
```
Appears to OOM on certain NPM operations. Not all, as ops like `npm install` complete successfully.

Let me know if you need any more information

**Comment by aem** [comment]
Whoops. Didn't do my due diligence. Appears to be an NPM issue, not Hyper. Disregard

**Comment by Alex0007** [comment]
Experiencing same error. How to resolve this?

**Comment by aem** [comment]
As I stated, it's an NPM issue: https://github.com/npm/npm/issues/12619#issuecomment-220176074
tl;dr: `npm install -g npm@latest`
